src/main/resources/api/swagger.yaml

```diff
@@ -1,74 +1,80 @@
 swagger: '2.0'
 info:
   title: Pazuzu registry
-  description: Generating Dockerfiles by sets of required functions
-  version: "0.0.4"
+  description: Generating Dockerfiles by sets of required functions.
+  version: "0.0.5"
 host: pazuzu.zalando.net
 schemes:
   - http
 produces:
   - application/json
 paths:
-  /api/resolved-features:
-    get:
-      summary: Resolve all features by name
-      description: |
-        Returns all the features with all its dependencies in a single array.
-      tags:
-        - Features
-      parameters:
-        - name: names
-          in: query
-          description: feature names.
-          required: true
-          type: array
-          items:
-            type: string
-      responses:
-        200:
-          description: An array of features
-          schema:
-            type: array
-            items:
-              $ref: '#/definitions/Feature'
-        default:
-          description: Unexpected error
-          schema:
-            $ref: '#/definitions/Error'
   /api/features:
     get:
-      summary: Get all features
+      summary: Get features
       description: |
-        Returns all the features without its dependencies.
+        Returns the features.
       tags:
         - Features
       parameters:
         - name: names
           in: query
-          description: value, that must present in feature name.
+          description: Exact names of the features the call should return.
           required: false
           type: array
+          maxItems: 128
```

Comment:
The main usage of the manes field is for resolution.  The idea is to limit the number of feature you can ask for a resolution to prevent to create an giant dependency graph.

In other hand you can create a meta feature that contains 1000 dependencies.